tools/monitoring/mysql/db_mysql.go

```diff
@@ -118,3 +118,45 @@ func PubsubMsgHandler(db *sql.DB, configURL, buildLogURL, jobname string, prNumb
 
 	return tx.Commit()
 }
+
+// CheckAlertCondition checks whether the given error pattern meets
+// the alert condition specified in config
+func CheckAlertCondition(errorPattern string, config *config.SelectedConfig, db *sql.DB) (bool, error) {
+	// the timestamp we want to start collecting logs
+	startTime := time.Now().Add(time.Minute * time.Duration(config.Period))
```

Comment:
I think the `startTime` variable can be removed by using `now()` in `CREATE VIEW...` statement.

Reply:
Why do we want to do it there though? There is computation to be performed, as you can see in the assignment and I don't think it is a good idea to put the computation logic in the query.

Reply:
Sorry, missed the period computation. Disregard my comment.